aws/resource_aws_elasticache_replication_group.go

```diff
@@ -276,6 +276,15 @@ func resourceAwsElasticacheReplicationGroup() *schema.Resource {
 				ForceNew: true,
 				Computed: true,
 			},
+			"user_group_ids": {
+				Type:       schema.TypeSet,
+				ConfigMode: 0,
+				Optional:   true,
+				Default:    nil,
+				Elem:       &schema.Schema{Type: schema.TypeString},
+				MaxItems:   1, //at the moment the aws sdk only supports 1 user group id to be associated with a cluster
+				Set:        schema.HashString,
+			},
```

Comment:
I am pretty sure this conflicts with the `auth_token` argument, so I think you should add `ConflictsWith` (here and then on the `auth_token` argument).

Like this:
```suggestion
			"user_group_ids": {
				Type:          schema.TypeSet,
				ConfigMode:    0,
				Optional:      true,
				Default:       nil,
				Elem:          &schema.Schema{Type: schema.TypeString},
				MaxItems:      1, //at the moment the aws sdk only supports 1 user group id to be associated with a cluster
				Set:           schema.HashString,
				ConflictsWith: []string{"auth_token"},
			},
```

Reply:
Ah thankyou, yes, i've also added the field to the auth_token block as well